src/components/Canvas/CanvasTemplate.vue

```diff
@@ -372,7 +351,7 @@ export default {
       // of the image if necessary
       if (deltaY < 0 && offsetY < this.windowHeight) {
         this.pageHeight = this.pageHeight - deltaY;
-        this.imageBlockController.forEach((imageBlock) => {
+        this.imageBlocks.forEach((imageBlock) => {
           imageBlock.centralImageTopPosition =
```

Comment:
should probably be updated to `imageBlock.centralImagePosition.top`
same for left below

Reply:
Right, I forgot this one. done